sdk/metrics/src/test/java/io/opentelemetry/sdk/metrics/internal/state/MetricStorageUtilsTest.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.internal.state;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.sdk.common.InstrumentationLibraryInfo;
+import io.opentelemetry.sdk.metrics.data.AggregationTemporality;
+import io.opentelemetry.sdk.metrics.data.MetricData;
+import io.opentelemetry.sdk.metrics.internal.aggregator.Aggregator;
+import io.opentelemetry.sdk.metrics.internal.aggregator.AggregatorHandle;
+import io.opentelemetry.sdk.metrics.internal.descriptor.MetricDescriptor;
+import io.opentelemetry.sdk.resources.Resource;
+import java.util.HashMap;
+import java.util.Map;
+import javax.annotation.Nullable;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class MetricStorageUtilsTest {
+
+  static final Attributes a = Attributes.of(AttributeKey.stringKey("a"), "a");
+  static final Attributes b = Attributes.of(AttributeKey.stringKey("b"), "b");
+  static final Attributes c = Attributes.of(AttributeKey.stringKey("c"), "c");
+  static final Attributes d = Attributes.of(AttributeKey.stringKey("d"), "d");
+  static final Attributes e = Attributes.of(AttributeKey.stringKey("e"), "e");
+  Map<Attributes, String> result;
+  Map<Attributes, String> toMerge;
```

Comment:
#nit: All these can be private.

Reply:
My preference for tests is to just omit access specifiers for brevity and clarity...but I can change it if folks generally think there's something wrong with that.

Reply:
We tend to keep things private in this repo, so yes, private would be good here.